src/frontend/components/node-item/node-attributes.ts

```diff
@@ -9,4 +9,11 @@ import {Property} from '../../../backend/utils';
 })
 export class NodeAttributes {
   @Input() private attributes: Array<Property>;
+
+  private show: boolean = false;
+
+  onShow(event) {
+    event.stopPropagation();
```

Comment:
Does it make sense to eat up the event here. Is it possible that a parent container might want to show something else off this event?